openreview/conference/templates/programchairWebfield.js

```diff
@@ -49,8 +49,13 @@ var getOfficialReviews = function(noteNumbers) {
     var ratingExp = /^(\d+): .*/;
 
     _.forEach(notes, function(n) {
-      var num = getNumberfromGroup(n.signatures[0], 'Paper');
-      var index = getNumberfromGroup(n.signatures[0], 'AnonReviewer');
+      if (_.startsWith(n.signatures[0], '~')) {
+        var num = getNumberfromGroup(n.invitation, 'Paper');
+        var index = n.signatures[0];
```

Comment:
I have to use the signature id as the index because the review can be signed as ~Melisa_Bok1.